src/devices/LuxBase.cpp

```diff
@@ -215,12 +216,14 @@ bool LuxBase::initTcp(Tcp::DisconnectFunction function, void* obj)	// , bool beV
 			infoMessage(m_longName + " UpsideDown is not active.", m_beVerbose);
 		}
 	}
+/*
+// some devices dont have the UpsideDown flag so just ignore this error
 	else
 	{
 		printError("LuxBase::initTcp(): ERROR: Failed to read UpsideDown flag, aborting.");
 		return false;
 	}
-
+*/
```

Comment:
Please don't commit commented code. Better print an informative message and return true.